### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by burtbeckwith**

### CODE REVIEW

File: grails-app/conf/BuildConfig.groovy [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 grails.project.dependency.resolution = {
 
-    inherits("global") {}
-    log "warn"
-    legacyResolve false
+    inherits 'global'
+    log 'warn'
 
     repositories {
+        mavenLocal()
```
Comment (mhurne) [inline grails-app/conf/BuildConfig.groovy]: We tend to prefer omitting use of local Maven repositories unless absolutely necessary, which we find is typically a short-lived need that doesn't need to be checked in. Would you be OK with removing this line?

File: grails-app/controllers/com/commercehub/grails/groovations/GroovationsController.groovy [hunk 2]
```diff
@@ -10,17 +10,12 @@ class GroovationsController {
...
-        def numExecutedMigrations = groovationsService.getCountOfExecutedScripts()
-        def areMigrationsExecuting = groovationsService.areMigrationsExecuting()
-
-        return [
-                pendingMigrations: pendingMigrations,
-                executedMigrations: executedMigrations,
-                pageSize: pageSize,
-                numExecutedMigrations: numExecutedMigrations,
-                areMigrationsExecuting: areMigrationsExecuting
+        [
```
Comment (mhurne) [inline grails-app/controllers/com/commercehub/grails/groovations/GroovationsController.groovy]: Would you be open to putting the explicit `return` keyword back in? We've run in to several nasty and difficult-to-diagnose bugs due to omitted `return` keywords in the past, and lean toward including them in most cases.

File: grails-app/services/com/commercehub/grails/groovations/GroovationsService.groovy [hunk 3]
```diff
@@ -2,16 +2,19 @@ package com.commercehub.grails.groovations
...
 import org.codehaus.groovy.grails.io.support.ClassPathResource
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
 import javax.annotation.PreDestroy
-import javax.annotation.Resource
 import java.util.concurrent.ExecutorService
 import java.util.concurrent.Executors
 
+@CompileStatic
```
Comment (mhurne) [inline grails-app/services/com/commercehub/grails/groovations/GroovationsService.groovy]: :+1:

File: src/groovy/com/commercehub/grails/groovations/MongoScriptExecutionJobRepository.groovy [hunk 4]
```diff
@@ -26,19 +29,14 @@ class MongoScriptExecutionJobRepository implements ScriptExecutionJobRepository
...
         job.id = dbObject.get(FIELD_ID)
     }
 
-    @Override
     boolean areJobsRunning() {
-        def runningJobsCount = collection.find(new BasicDBObject([
-                (FIELD_STATUS): JobStatus.RUNNING.name()
-        ])).count()
-        return runningJobsCount as boolean
+        collection.find(new BasicDBObject([(FIELD_STATUS): JobStatus.RUNNING.name()])).count()
```
Comment (mhurne) [inline src/groovy/com/commercehub/grails/groovations/MongoScriptExecutionJobRepository.groovy]: Would you be open to putting the explicit `return` keyword back in here as well?

**Comment by mhurne** [comment]
This is great stuff, Burt. Thanks! I only have a few minor comments that have to do with our preferences around the use of Maven local and explicit `return` keywords.

@da14purc: Do you want to take a look as well?

**Comment by burtbeckwith** [comment]
Please treat all changes as suggestions and revert whatever you want.

**Comment by mhurne** [comment]
Will do. Thanks!

**Comment by da14purc** [comment]
@mhurne I'll take a look at these changes Monday or Tuesday night.

**Comment by mhurne** [comment]
As this project is now deprecated, CommerceHub will not be contributing further on this ticket.  Please feel free to work on it further yourself under the terms of the project license if desired.

**Pull request closed without merging**